tests/testthat/helper-for-tests.R

```diff
@@ -1,22 +1,27 @@
-
-getTestDataFilePath <- function(fileName) {
-  dataPath <- file.path(getwd(), "..", "data", fsep = .Platform$file.sep)
-  file.path(dataPath, fileName, fsep = .Platform$file.sep)
-}
-
-getSimulationFilePath <- function(simulationName) {
-  getTestDataFilePath(paste0(simulationName, ".pkml"))
-}
-
-# Helper function to load a model easily. In the test environment, we do not want to load from cache by default. Instead
-# new instances should be created unless specifically specified otherwise
-loadTestSimulation <- function(simulationName, loadFromCache = FALSE, addToCache = TRUE) {
-  simFile <- getSimulationFilePath(simulationName)
-  sim <- loadSimulation(simFile, loadFromCache = loadFromCache, addToCache = addToCache)
-}
-
-executeWithTestFile <- function(actionWithFile) {
-  newFile <- tempfile()
-  actionWithFile(newFile)
-  file.remove(newFile)
-}
+
+getTestDataFilePath <- function(fileName) {
+  dataPath <- file.path(getwd(), "..", "data", fsep = .Platform$file.sep)
+  file.path(dataPath, fileName, fsep = .Platform$file.sep)
+}
+
+getExtDataFilePath <- function(fileName) {
```

Comment:
Another helping function so we can create paths to files stored in `extdata`, soo we can avoid duplication of files for tests and examples.

Reply:
I am talking only for files that are actually duplicated. We sometimes use the same files for test and examples.

Of course, we still should have files that are only for testing. This function should not replace `getTestDataFilePath`.

Reply:
ok